.jshintrc

```diff
@@ -0,0 +1,27 @@
+{
+	"node": true, 
+
+	"browser": true, 
```

Comment:
Since yadda only exists as a CLI tool, we probably don't need the `browser: true` setting, right?